miner/worker.go

```diff
@@ -1167,6 +1246,8 @@ func (w *worker) fillTransactions(interrupt *atomic.Int32, env *environment) err
 			return err
 		}
 	}
+	// append a DAG tx at the end of the block
+	w.appendTxDAG(env)
```

Comment:
If there has MEV bundles, this tx may not the last one?

Reply:
Yes, indeed.